modules/i3s/src/lib/parsers/parse-i3s-tile-content.js

```diff
@@ -333,3 +335,86 @@ function offsetsToCartesians(vertices, metadata = {}, cartographicOrigin) {
 
   return positions;
 }
+
+/**
+ * Makes PBR material from material definition
+ * @returns {object}
+ */
+function makePbrMaterial(materialDefinition, texture) {
```

Comment:
```suggestion
function makePbrMaterial(i3sMaterial, texture) {
```

Reply:
It's exactly `materialDefinition` from the tileset metadata https://github.com/Esri/i3s-spec/blob/master/docs/1.7/materialDefinitions.cmn.md